config/dynamicconfig/development_es.yaml

```diff
@@ -13,7 +13,7 @@ frontend.validSearchAttributes:
       StartTime: "Int"
```

Comment:
the index_template.json has this as a long. Same with executiontime and closetime

Reply:
Looks like proto definition only defines the following four types:
```
message Field {
    temporal.server.api.enums.v1.FieldType type = 1;
    oneof data {
        string string_data = 2;
        int64 int_data = 3;
        bool bool_data = 4;
        bytes binary_data = 5;
    }
}
```

Reply:
hmm, then maybe we should just fix up index_template to use integer then

Reply:
Updated.